packages/internal/src/generate/plugins/rw-typescript-resolvers/index.ts

```diff
@@ -0,0 +1,142 @@
+// Copied from
+// https://github.com/dotansimha/graphql-code-generator/blob/5efc9433431728f7bcb35d33b6d9e29c4313d4f0/packages/plugins/typescript/resolvers/src/index.ts
+// and then modified
+
+import {
+  Types,
+  PluginFunction,
+  addFederationReferencesToSchema,
+  getCachedDocumentNodeFromSchema,
+  oldVisit,
+} from '@graphql-codegen/plugin-helpers'
+import {
+  TypeScriptResolversPluginConfig,
+  plugin as basePlugin,
+} from '@graphql-codegen/typescript-resolvers'
+import { GraphQLSchema } from 'graphql'
+
+import { RwTypeScriptResolversVisitor } from './visitor.js'
+
+export const plugin: PluginFunction<
+  TypeScriptResolversPluginConfig,
+  Types.ComplexPluginOutput
+> = (
+  schema: GraphQLSchema,
+  _documents: Types.DocumentFile[],
+  config: TypeScriptResolversPluginConfig
+) => {
+  const imports = []
+  if (!config.customResolveInfo) {
+    imports.push('GraphQLResolveInfo')
+  }
+
+  const indexSignature = config.useIndexSignature
+    ? [
+        'export type WithIndex<TObject> = TObject & Record<string, any>;',
+        'export type ResolversObject<TObject> = WithIndex<TObject>;',
+      ].join('\n')
+    : ''
+  const defsToInclude: string[] = []
+
+  const transformedSchema = config.federation
+    ? addFederationReferencesToSchema(schema)
+    : schema
+  const visitor = new RwTypeScriptResolversVisitor(config, transformedSchema)
```

Comment:
This is pretty much the only important thing in this file. I'm using our own Visitor here.

All other changes are about removing code that we don't need with our specific setup.